### PR TITLE
Add CCPA links to configs

### LINKED ISSUE #5588: Add CCPA link to all footers
**Is your feature request related to a problem? Please describe.**
As a part of our response to CCPA, we need to add a link to all our footers.

**Describe the solution you'd like**
The link text should be `AdChoices / Do Not Sell My Info`
The link destination should be `https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/`
The link text should be in English on all services (with an English lang attribute)
1. Update [`psammead-sitewide-links`](https://github.com/bbc/psammead/tree/latest/packages/components/psammead-sitewide-links) to accept an optional lang attribute on its anchors )`<a>`.
2. Pull the updated `psammead-sitewide-links` into Simorgh and add an English lang attribute to relevant link - `AdChoices / Do Not Sell My Info`.

**Describe alternatives you've considered**
N/A

**Testing notes**
- These changes will require a dev a11y swarm.
- The snapshot could record the new lang= attribute.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by FK78**
Resolves #5588

**Overall change:** _Added CCPA links to service configs._

**Code changes:**
- _Added in CCPA links to the `news` service config._
- _Updated snapshots to include new CCPA link._
- _Bumped `psammead-sitewide-links`._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by rhenshaw56 — APPROVED** [review]

File: src/app/lib/config/services/news.js [hunk 1]
```diff
@@ -202,6 +202,12 @@ export const service = {
           href: 'https://www.bbc.com/contact/',
           text: 'Contact the BBC',
         },
+        {
+          href:
+            'https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
+          text: 'AdChoices / Do Not Sell My Info',
+          lang: 'en-GB',
```
Comment (andrewscfc) [inline src/app/lib/config/services/news.js:209]: I think this link should be on every service's config: 

> The link text should be in English on all services (with an English lang attribute)

**Review by andrewscfc — CHANGES_REQUESTED** [review]
I'm not sure this meets the requirements on the ticket

File: src/app/lib/config/services/news.js (hunk 1 above)
Comment (FK78) [reply src/app/lib/config/services/news.js:209]: The link has been added to all services.

File: src/app/lib/config/services/arabic.js [hunk 2]
```diff
@@ -205,6 +205,12 @@ export const service = {
           href: 'https://www.bbc.com/arabic/institutional-37731355',
           text: 'اتصل بـ بي بي سي',
         },
+        {
+          href:
+            'https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
+          text: 'AdChoices / Do Not Sell My Info',
+          lang: 'en-GB',
```
Comment (rhenshaw56) [inline src/app/lib/config/services/arabic.js:212]: @FK78 I think the lang: 'en-GB' should just apply to English and Scotland.

I also think the text `'AdChoices / Do Not Sell My Info'` might be translated for non English services cc @jimjohnsonrollings

**Comment by jimjohnsonrollings** [comment]
Nope. English for everyone for this thanks.

**Review by andrewscfc — APPROVED** [review]
Thanks for addressing this @FK78 🙂

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**